x-pack/plugins/security_solution/public/detections/components/callouts/ml_job_compatibility_callout/translations.tsx

```diff
@@ -13,28 +13,27 @@ import { MlJobCompatibilityLink } from '../../../../common/components/links_to_d
 export const ML_JOB_COMPATIBILITY_CALLOUT_TITLE = i18n.translate(
   'xpack.securitySolution.detectionEngine.mlJobCompatibilityCallout.messageTitle',
   {
-    defaultMessage: 'Your ML jobs may be incompatible with your data sources and/or ML rules',
+    defaultMessage:
+      'New updates are available for your ML jobs, please see documentation before updating Detection Rules to ensure continued coverage',
   }
 );
 
 export const MlJobCompatibilityCalloutBody = () => (
   <FormattedMessage
     id="xpack.securitySolution.detectionEngine.mlJobCompatibilityCallout.messageBody"
-    defaultMessage="{summary} Related documentation: {docs}"
+    defaultMessage="{summary} Documentation: {docs}"
     values={{
       summary: (
         <p>
           <FormattedMessage
             id="xpack.securitySolution.detectionEngine.mlJobCompatibilityCallout.messageBody.summary"
-            defaultMessage="Machine learning rules specify ML jobs that in
-            turn have dependencies on data fields populated by the Elastic
-            beats and agent integrations that were current when the ML job
-            was created. New ML jobs, prefixed with V2, have been updated to
-            operate on now-current ECS fields. If you are using multiple
-            versions of beats and agents, you need to create new machine
-            learning rules that specify the new ML (V2) jobs, and enable them
-            to run alongside your existing machine learning rules, in order
-            to ensure continued rule coverage."
+            defaultMessage="New V3 Machine Learning jobs have been released and
+            the corresponding Detections Rules have been updated to use these
+            new jobs. You are currently running one or more V1/V2 jobs and
+            action is required to ensure continued coverage before updating
+            your Detection Rules. Please see the below documentation for
+            instructions on how to keep using the V1/V2 jobs, and how
+            to start using the new V3 jobs."
```

Comment:
Possible edit -- let me know if I took any liberties that need to be further revised. I was going to bold/italic the word "before" in the middle ("...you may need to duplicate or create new rules _before_ updating..."), but didn't want to fuss with the `FormattedMessage` syntax while we're still just iterating on the content.
```suggestion
            defaultMessage="New V3 machine learning jobs have been released, 
            and the latest corresponding prebuilt detection rules now use these 
            new ML jobs. You're currently running one or more V1/V2 jobs, which 
            only work with legacy prebuilt rules. To ensure continued coverage using 
            V1/V2 jobs, you may need to duplicate or create new rules before 
            updating your Elastic prebuilt detection rules. Check the documentation 
            below for instructions on how to keep using the V1/V2 jobs, and how to 
            start using the new V3 jobs."
```

Reply:
Used suggested copy in modal here: https://github.com/elastic/kibana/pull/128334/files#diff-bfb1a4fcf236f6e864bdc1305e01021f719f724aa61ed5d7fccd557489f3579bR50-R57